cibuildwheel/typing.py

```diff
@@ -42,3 +42,9 @@
 
 PlatformName = Literal["linux", "macos", "windows"]
 PLATFORMS: Final[set[PlatformName]] = {"linux", "macos", "windows"}
+
+
+class GenericPythonConfiguration(Protocol):
+    @property
+    def identifier(self) -> str:
+        pass
```

Comment:
```suggestion
        ...
```

Typing uses `...` conventionally.